pkg/codegen/python/gen.go

```diff
@@ -1747,26 +1747,41 @@ func (mod *modContext) genMethods(w io.Writer, res *schema.Resource) {
 		methodName := PyName(method.Name)
 		fun := method.Function
 
```

Comment:
There'll be some code in this file where we emit the version of the SDK that the generated SDK depends on. We'll need to bump that to the version we released with the core SDK changes that included `call_single`/`extract_single_value` etc., ideally _only_ if we see a method that needs that feature.